packages/dotlottie-js/src/common/utils.ts

```diff
@@ -635,10 +635,16 @@ export async function inlineImageAssets(
     }
   }
 
-  const unzippedImages = await getImages(dotLottie, (file) => imagesMap.has(file.name));
+  const unzippedImages = await getImages(dotLottie, (file) => {
+    // Images inside the .lottie stored as 'images/[image_name]'
+    // Images inside the map are stored as '[image_name]' so we need to remove the 'images/' prefix
```

Comment:
Nice catch! 👏🏼

I believe the `getImages` filter function should expect `file.name` to always represent the image file name. This would prevent confusion when users begin to use it. Therefore, I'd recommend updating the filter methods in `getImages`, `getThemes`, and `getAnimations` to remove the path prefixes like `animations/`, `themes/`, or `images/` when file.name is passed to the filter function.